src/utils.cc

```diff
@@ -162,10 +162,12 @@ namespace ctranslate2 {
   }
 
   void set_num_threads(size_t num_threads) {
-#ifdef _OPENMP
+#if defined(_OPENMP)
     if (num_threads == 0)
       num_threads = read_int_from_env("OMP_NUM_THREADS", 4);
     omp_set_num_threads(num_threads);
+#elif defined(CT2_WITH_RUY)
```

Comment:
I think you should move this to a separate `#if` block since OpenMP and Ruy can be used at the same time.